packages/venia-concept/src/components/ProductFullDetail/ProductFullDetail.js

```diff
@@ -3,6 +3,7 @@ import { arrayOf, bool, func, number, shape, string } from 'prop-types';
 import { Price } from '@magento/peregrine';
 
 import classify from 'src/classify';
+import Button from 'src/button';
```

Comment:
Will be better with this `import Button from 'src/components/Button';`

Reply:
Good catch. This line introduced a bug, which I had to fix with #185.

The bug made its way into `master` because I made changes to this PR after it was already approved. We'll be tightening up our processes to not let this kind of bug slip past again.